services/director-v2/tests/unit/conftest.py

```diff
@@ -313,11 +326,15 @@ def fake_service_specifications(faker: Faker) -> dict[str, Any]:
 
 
 @pytest.fixture
-def mocked_catalog_service_fcts(
+def mocked_catalog_service_api(
```

Comment:
and probably should be here.. that happens sometimes with the sorting or black not sure which one

Reply:
the title groups a set of fixtures